packages/xy-chart/src/series/BoxPlotSeries.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Group } from '@vx/group';
+import { BoxPlot } from '@vx/stats';
+import themeColors from '@data-ui/theme/build/color';
+
+import { callOrValue, isDefined } from '../utils/chartUtils';
+
+import { boxPlotSeriesDataShape } from '../utils/propShapes';
+
+const propTypes = {
```

Comment:
I think for `renderTooltip` you'll have to add `onMouseMove`, `onMouseLeave`, and `onClick` handlers here

Reply:
yes! done